src/components/manage/History/History.jsx

```diff
@@ -116,22 +116,46 @@ class History extends Component {
     this.props.revertHistory(getBaseUrl(this.props.pathname), value);
   }
 
-  /**
-   * Render method.
-   * @method render
-   * @returns {string} Markup for the component.
-   */
-  render() {
+  processHistoryEntries = () => {
+    // Getting the history entries from the props
+    // No clue why the reverse(concat()) is necessary
```

Comment:
I guess someone just didn't want to figure out how to write a for loop to go through the entries in reverse order?